wgsl/index.bs

```diff
@@ -1039,15 +1039,41 @@ switch_stmt
   : SWITCH paren_rhs_stmt BRACKET_LEFT switch_body+ BRACKET_RIGHT
 
 switch_body
-  : CASE const_literal COLON BRACKET_LEFT case_body BRACKET_RIGHT
+  : CASE case_selectors COLON BRACKET_LEFT case_body BRACKET_RIGHT
   | DEFAULT COLON BRACKET_LEFT case_body BRACKET_RIGHT
 
+case_selectors
+  : const_literal (COMMA const_literal)*
+
 case_body
   :
   | statement case_body
   | FALLTHROUGH SEMICOLON
 </pre>
 
+A switch statement transfers control to one of a set of case clauses, or to a default clause,
+depending the evaluation of a selector expression of a scalar integer type.
+
+If the selector value equals a value in a case selector list, then control is transferred to
+the body of that case clause.
+If the selector value does not equal any of the case selector values, then control is
+transferred to the `default` clause.
+
+Each switch statement must have exactly one default clause.
```

Comment:
Is this really a `must`? Shouldn't it be `may`? The default case isn't required is it?

Reply:
It's a must because of #714